server/core/store.go

```diff
@@ -279,6 +294,26 @@ func (s *StoreInfo) RegionScore(highSpaceRatio, lowSpaceRatio float64, delta int
 	return score / math.Max(s.GetRegionWeight(), minWeight)
 }
 
+func (s *StoreInfo) regionScoreV2(delta int64, deviation int) float64 {
+	A := float64(float64(s.GetAvgAvailable())-float64(deviation)*float64(s.GetAvailableDeviation())) / gb
+	C := float64(s.GetCapacity()) / gb
+	R := float64(s.GetRegionSize() + delta)
+	var (
+		K, M float64 = 1, 256 // Experience value to control the weight of the available influence on score
+		F    float64 = 20     // Experience value to prevent some nodes from running out of disk space prematurely.
+	)
+
+	var score float64
+	if A >= C || C < 1 {
+		score = R
+	} else if A > F {
+		score = (K + M*(math.Log(C)-math.Log(A-F+1))/(C-A+F-1)) * R
```

Comment:
How about adding some comments or some reference to explain why the function look like this?

Reply:
good idea.